tests/test_wave_06.py

```diff
@@ -42,22 +42,18 @@ def test_post_task_ids_to_goal_already_with_goals(client, one_task_belongs_to_on
     assert len(Goal.query.get(1).tasks) == 2
 
 
-@pytest.mark.skip(reason="No way to test this feature yet")
+# @pytest.mark.skip(reason="No way to test this feature yet")
 def test_get_tasks_for_specific_goal_no_goal(client):
     # Act
     response = client.get("/goals/1/tasks")
     response_body = response.get_json()
 
     # Assert
     assert response.status_code == 404
+    assert response_body == {"message": "Goal not found"}
```

Comment:
Nice work with this test completion! This assertion is a great way to validate functionality is working as expected